asreview/webapp/src/ProjectComponents/AnalyticsComponents/ProgressDensityChart.js

```diff
@@ -353,10 +356,13 @@ export default function ProgressDensityChart(props) {
                               <Typography variant="subtitle2">
                                 Persistent irrelevant records
```

Comment:
```suggestion
                                Irrelevant records only
```

Reply:
I was searching for an alternative to 'persistent' for its negative connotation.

Reply:
What about "Continued irrelevant records"?

Reply:
in the first chart, there are also irrelevant records, and often many more compared to relevant records...